data/transaction.go

```diff
@@ -165,6 +165,13 @@ type ExtendedApiSmartContractResult struct {
 	Used bool `json:"-"`
 }
 
+// ExtendedApiTransactionResult extends the original transaction.ApiTransactionResult with extra fields
+//   ProcessedStatus: will contain the transaction status after local processing
+type ExtendedApiTransactionResult struct {
+	*transaction.ApiTransactionResult
+	ProcessedStatus transaction.TxStatus `json:"processingStatus,omitempty"`
```

Comment:
might add a TODO to move it into `mx-chain-core-go` for `rc/v1.6.0`

Reply:
done